www/dusv03/styles/taskstyle.css

```diff
@@ -0,0 +1,128 @@
+*{
+    box-sizing: border-box;
+}
+
+.part1 {
+  font-family: sans-serif;
+}
+
+.part2 {
+    color: red;
+}
+
+.part3 {
+    color: rgb(47, 207, 47);
+}
+
+.part4 {
+    color: #5555FF;
+}
+
+.part5 {
+    font-size: 18px;
+}
+
+.part6 {
+    font-size: 24px;
+    font: italic;
+}
+
+.part7 {
+    color: white;
+    background-color: brown;
+}
+
+.part8 {
+    text-align: center;
+    line-height: 32px;
+}
+
+.part9 {
+    text-align: justify;
+    text-transform: uppercase;
+}
+
+.part10 {
+   height: 300px; 
+   width: 400px;
+   border: 1px solid purple;
+   padding: 30px;
+}
+
+.part11 {
+    height: 200px; 
+    width: 500px;
+    margin: 100px;
+    background-color: rgb(200,21,21)
+}
+
+.part12{
+    color: white;
+}
+
+.part13 {
+    width: 100%;
+    height: 150px;
+    background-color: blue;
+    padding: 50px;
+    border: 10px solid yellow;
+}
+
+.part14{
+    color: white;
+    margin-top:50px;
+}
+
+.part15 {
+    height: 200px; 
+    width: 500px;
+    background-color: green;
+    opacity: 0.5;
+    
+}
+
+.part16 {
+    height: 400px; 
+    width: 300px;
+    background-color: lightblue;
+    position: relative;
+}
+
+.part18 {
+    height: 50px; 
+    width: 50px;
+    background-color: pink;
+    position: absolute;
+    top: 50px;
+    right:10px;
+}
+
+.part20 {
+    height: 100px; 
+    width: 300px;
+}
+
+.part20:active{
+    color: blue;
+}
+
+.part21 {
+    height: 50px; 
+    width: 200px;    
+}
+
+.part21:hover{
+    background-color: brown;
+}
+
+
+.part22 {
+    height: 500px; 
+    width: 200px; 
+    background-color: #4444eb;
+    margin-top: 200px;
+}
+
+.part22:hover{
+    cursor:pointer;
```

Comment:
vlastnost cursor: pointer nemusi byt v :hover protoze jiz sama implikuje to, ze element musi byt hovered aby to melo efekt

Reply:
je tedy potřeba tento bod opravovat ? resp. nevím jak bych to měl upravit

Reply:
opravit ci ne je na tobe, jde o to abys o tom vedel, ja bych usetril misto a nekomplikoval to,